rust/Earthfile

```diff
@@ -70,43 +91,41 @@ get-jq:
     FROM alpine:3.18.3
     ARG USERARCH
     ARG version=1.7
-    RUN wget -O jq https://github.com/jqlang/jq/releases/download/jq-${version}/jq-linux-${USERARCH} && \
+    RUN wget -O jq https://github.com/jqlang/jq/releases/download/jq-${version}/jq-linux-${USERARCH}; \
         chmod +x jq
     SAVE ARTIFACT jq
 
 # INSTALL_CARGO_SWEEP installs cargo-sweep if it doesn't exist already.
 INSTALL_CARGO_SWEEP:
     COMMAND
-    DO +RUN_WITH_CACHE --command="set -e;
-        if [ ! -f \$CARGO_HOME/bin/cargo-sweep ]; then
-          cargo install cargo-sweep --root \$CARGO_HOME;
-        fi;"
+    RUN if [ ! -f $CARGO_HOME/bin/cargo-sweep ]; then \
```

Comment:
Why no longer caching this?

Reply:
Before it was also being installed in the build environment (notice the `$CARGO_INSTALL_ROOT` override).

This change moves it to +INIT so, the user can run this before copying files from their context, and this installation is cached even when source files change.

The reason to move this out of the cache was to avoid the scenario of BK garbage collecting these files, while the RUN command installing it still kept cached. 

We could have handled this case by always checking the command exists before using it (in the same RUN command) but overall I thought that moving the installation to the init phase would make things cleaner.